Subject: [PATCH] Update
---
Index: XssWithRemediation.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/XssWithRemediation.java b/XssWithRemediation.java
--- a/XssWithRemediation.java	(revision 74f30ca5540fa6ed89cd5023cfb749911808a4b5)
+++ b/XssWithRemediation.java	(date 1689178290431)
@@ -49,7 +49,7 @@
                 pw.println(logLine);
             }
         } catch (FileNotFoundException e) {
-            log.error("Error occurred while writing the logfile", e);
+            log.error("Error occurred while writing to the logfile", e);
         }
         return "";
     }
